warehouse/templates/manage/organization/teams.html

```diff
@@ -24,7 +24,7 @@
   <h2>{% trans team_count=organization.teams|length %}Teams ({{ team_count }}){% endtrans %}</h2>
 
   <div class="team-list">
-    {% for team in organization.teams %}
+    {% for team in organization.teams|sort(attribute="name") %}
```

Comment:
<!-- Thoughts represent an idea that popped up from reviewing. These comments are non-blocking by nature, but they are extremely valuable and can lead to more focused initiatives and mentoring opportunities.  -->
**thought:** ‏ While this works, is there a way to send more of the responsibility upstream to the model, and keep the template "simpler"? Likely not a real tradeoff until any organization has a large volume of teams that would cost more to sort in Python than sorting in the Postgres query that returns them already-in-order. Not blocking by any means.

Reply:
@miketheman Shifted sorting responsibility upstream to `Organization` model in commit [7365596](https://github.com/pypi/warehouse/pull/12223/commits/7365596d52a72ab487770a0a07fab73add6297af).